### ISSUE TITLE
aria-required should be allowed on input type="file"

### ISSUE THREAD

**Issue by ghost** [w3c/html-aria#234]
Hi,

The [HTML validator](https://validator.w3.org/nu/#textarea) is telling me that using an `aria-required` attribute on an `<input type="file">` is an error. In the ARIA in HTML specification, I can read that this field can accept all "[Global aria-* attributes](https://w3c.github.io/html-aria/#index-aria-global)". `aria-required` is not on the list.

But, if I read the [aria-required specification](https://w3c.github.io/html-aria/#att-required), I can read "Use the `aria-required` attribute on any element that is allowed the `required` attribute in HTML."

I can use a `required` attribute on an `<input type="file">` so I should be able to use an `aria-required` attribute too.

**Comment by scottaohara** [comment]
hi @juliemoynat-tanaguru, this looks like an issue with the validator, not this specification.  `aria-required` should not throw an error on a file input, and doesn't for other input types when testing in the validator.

**Comment by ghost** [comment]
Hi @scottaohara, shouldn't [the specification for the `input type="file"`](https://w3c.github.io/html-aria/#el-input-file) specify that the use of the `aria-required` attribute is allowed (just like the `required ` attribute) as it's not a global aria-* attribute?

**Comment by scottaohara** [comment]
No. The specification has [`required`](https://w3c.github.io/html-aria/#att-required) which covers `aria-required` allowances.  The only input that currently states that `aria-required` is allowed beyond global aria attributes is password.  But that's an unnecessary call out per what I just referenced.

**Comment by ghost** [comment]
Ok. So I will create a ticket for the validator. Thank you
